src/ducks/user.duck.js

```diff
@@ -317,7 +318,17 @@ export const fetchCurrentUser = (params = null) => (dispatch, getState, sdk) =>
 
   const parameters = params || {
     include: ['profileImage', 'stripeAccount'],
-    'fields.image': ['variants.square-small', 'variants.square-small2x'],
+    'fields.image': ['variants.avatar', 'variants.avatar2x'],
```

Comment:
Since this is in user.duck.js, does this affect the ProfileSettingsPage and ProfilePage too? I thought that you would only add new variants.

Reply:
Indeed, is the square-small2x needed? Would just adding square-small to that list do?

Reply:
@lyyder if ProfileSettingPage gets the image from user.duck.js, then this needs also retina version (i.e. both 'variants.square-small', 'variants.square-small2x'). 

So, if that's the case, then I'd suggest that you just add those new avatar variants (for Topbar) as extra. The components & browser then decide which of the image they are going to use.